content/en/about/feature-stages/index.md

```diff
@@ -48,6 +48,7 @@ Below is our list of existing features and their current phases. This informatio
 | Enabling custom filters in Envoy | Alpha
 | CNI container interface | Alpha
 | [Sidecar API](/docs/reference/config/networking/sidecar/) | Beta
+| [Configuring Gateway Network Topology](/docs/ops/configuration/traffic-management/network-topologies/) | Experimental
```

Comment:
`Experimental` is not a valid feature status. I would strongly encourage just removing this line until the code is Alpha, Beta, or Stable. I have submitted PR: https://github.com/istio/community/pull/320 and will add this topic to the TOC agenda.

https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md

Reply:
I don't think this needs listing here. Any reason @jacob-delgado you added it?

Reply:
I was not familiar with feature stages. I've removed it.